helm/external-dns-app/templates/_helpers.tpl

```diff
@@ -136,7 +136,7 @@ Validate that the provider makes sense
 don't expose that value to the user in the error message.
 */}}
 {{- define "validateValues.provider" -}}
-{{- if and (ne .Values.provider "aws") (ne .Values.provider "azure") (ne .Values.provider "inmemory") -}}
+{{- if and (ne .Values.provider "aws") (ne .Values.provider "azure") (ne .Values.provider "vmware") (ne .Values.provider "inmemory") -}}
 external-dns: provider
     Incorrect value provided. Valid values are either 'aws' or 'azure'.
```

Comment:
```suggestion
    Incorrect value provided. Valid values are either 'aws', 'azure', 'vmware' and 'inmemory'.
```

Reply:
also god spot. we don't want to expose the `inmemory` provider though as that's only there for CI tests: https://github.com/giantswarm/external-dns-app/blob/master/integration/templates/external_dns_values.go#L7